package.json

```diff
@@ -97,23 +97,24 @@
     "prismjs": "1.23.0",
     "reflect-metadata": "0.1.8",
     "rxjs": "^6.5.4",
-    "zone.js": "~0.10.2"
+    "tslib": "^2.0.0",
+    "zone.js": "~0.10.3"
   },
   "devDependencies": {
     "@angular-builders/custom-webpack": "^8.4.1",
-    "@angular-devkit/build-angular": "~0.901.0",
-    "@angular/cli": "^9.1.0",
-    "@angular/compiler-cli": "^9.1.0",
-    "@ngneat/spectator": "^5.3.1",
+    "@angular-devkit/build-angular": "~0.1102.4",
+    "@angular/cli": "^11.2.4",
+    "@angular/compiler-cli": "^11.2.5",
+    "@ngneat/spectator": "^5.13.4",
     "@nrwl/cypress": "^9.1.2",
     "@nrwl/workspace": "^9.1.2",
-    "@types/jasmine": "^3.5.9",
+    "@types/jasmine": "~3.6.0",
     "@types/jasminewd2": "^2.0.8",
     "@types/jest": "^25.1.4",
     "@types/node": "^10.17.17",
     "@types/webpack": "^4.41.7",
     "chokidar": "^3.3.1",
-    "codelyzer": "^5.0.1",
+    "codelyzer": "^6.0.0",
     "copy-webpack-plugin": "^5.1.1",
     "core-js": "^2.6.11",
```

Comment:
We only import core-js in a single test file - it's a polyfil for modern js features, so I have a hard time believing it is necessary. Kill it with 🔥?

Reply:
![giphub](https://media1.giphy.com/media/e0dU1Hy8pbGO4/giphy.gif)

Let's see what happens...